Spokestack/NLUTensorflowSlotParsers.swift

```diff
@@ -14,77 +14,79 @@ internal struct NLUTensorflowSlotParser {
     /// Parse the classification input and predicted tag labels into a structured [intent : Slot] dictionary.
     /// - Note: This operation effectively truncates the tag labels by the input size, ignoring all labels outside the input token count.
     /// - Parameters:
-    ///   - taggedInput: A zip of tokenized inputs and classification tag labels.
+    ///   - tags: An array of classification tag labels.
     ///   - intent: The predicted intent.
-    internal func parse(taggedInput: Zip2Sequence<[String], [String]>, intent: NLUTensorflowIntent, encoder: BertTokenizer) throws -> [String:Slot] {
+    internal func parse(tags: [String], intent: NLUTensorflowIntent, encoder: BertTokenizer, encodedTokens: EncodedTokens) throws -> [String:Slot] {
         
+        guard let encodedTokensByWhitespaceIndex = encodedTokens.encodedTokensByWhitespaceIndex else {
+            throw NLUError.tokenizer("NLU model tokenizer failed to correctly encode utterance because encodedTokensByWhitespaceIndex is nil.")
```

Comment:
It's good form to guard inside of each function that uses an Optional instead of assuming implicit dependencies, but this error message is definitely copy-pasta.

Reply:
Yeah...what I meant was that I don't think that field should be an Optional. If you don't have that information, something went wrong during tokenization, and classification shouldn't be attempted.